### PR TITLE
Update CATS to include context route path

### PR DESCRIPTION
**Pull request by shashwathi**
This PR updates CATS to use CF CLI for testing context route path.

Pivotal tracker story: https://www.pivotaltracker.com/story/show/106693894

Shash & Chris
Routing Team
Cloudfoundry

### CODE REVIEW

**Comment by Amit-PivotalLabs** [comment]
@shashwathi `CreateRoute` should no longer return anything.

**Comment by shashwathi** [comment]
@Amit-PivotalLabs Thank you for prompting that. Fixed it.

File: routing/routing_suite_test.go [hunk 1]
```diff
@@ -90,49 +104,37 @@ func GetAppGuid(appName string) string {
...
+		"--hostname", hostname,
+		"--path", contextPath,
+	).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
 }
 
-func CreateRoute(hostName, contextPath, spaceGuid, domainGuid string) string {
-	jsonBody := "{\"host\":\"" + hostName + "\", \"path\":\"" + contextPath + "\", \"domain_guid\":\"" + domainGuid + "\",\"space_guid\":\"" + spaceGuid + "\"}"
-	session := cf.Cf("curl", "/v2/routes", "-X", "POST", "-d", jsonBody).Wait(CF_PUSH_TIMEOUT)
-	routePostResponseBody := session.Out.Contents()
+func GetRouteGuid(hostname, path string) string {
```
Comment (robdimsdale) [inline routing/routing_suite_test.go]: Does this method need to be exported? I did a quick search and it doesn't look like it's used outside the package:

```
± |master ↑2 ✓| → ag GetRoute
routing/route_services_test.go
159:	routeGuid := GetRouteGuid(hostname, "")
173:	routeGuid := GetRouteGuid(hostname, "")
208:	routeGuid := GetRouteGuid(hostname, "")

routing/routing_suite_test.go
126:func GetRouteGuid(hostname, path string) string {
```

**Comment by shashwathi** [comment]
@robdimsdale 

Instructions to test this route services
- Requires diego deployment
- Add `properties.acceptance_tests.include_route_services` to true in manifest
- CF cli version v6.15 in cf-release

**Comment by Amit-PivotalLabs** [comment]
Hi @shashwathi 

Could you update the minCLIVersion in the cli_compatibility_check test, and also add the deaUnsupportedTag to the test(s) that required Diego?

I will update the [contributing](https://github.com/cloudfoundry/cf-acceptance-tests#contributing) docs to mention these things.

Thanks,
Amit

**Comment by shashwathi** [comment]
@Amit-PivotalLabs : 
- Updated route-service suite with deaUnsupported tag 
- Updated cli_compatibility_test

Regards
Shash

**Comment by robdimsdale** [comment]
Merged, thanks.

Rob && @APShirley

**Pull request merged**